dapps/coin_flip.glow

```diff
@@ -1,7 +1,12 @@
 #lang glow
 // TODO: insert escrow in a compiler pass
 @interaction([A, B])
-let coinFlip = (wagerAmount, escrowAmount) => {
+let coinFlip = (wagerAmount : UInt256, escrowAmount : UInt256)
+
+ // required to ensure that the withdraw! statements do not overflow.
+ | (wagerAmount / 2) < MaxUInt256 - escrowAmount
```

Comment:
Note that I'm not 100% sure this constraint is correct -- there might be an off by one or something here, and I'm too tired to think through this methodically. But if it's wrong, the solver should complain that it still can't be sure there's no overflow.